apps/prairielearn/src/batched-migrations/20250318174636_course_instance_usages__submissions__backfill_v3.sql

```diff
@@ -0,0 +1,60 @@
+-- BLOCK delete_old_usages
+DELETE FROM course_instance_usages
+WHERE
+  type = 'Submission'
+  AND date >= $START_DATE
+  AND date < $END_DATE;
```

Comment:
I think this will be a bit slow, but I believe that it won't block INSERTs so we should be fine.